data/categories.txt

```diff
@@ -1338,7 +1345,7 @@ sw:Uzuri saluni
 fa:سالن زیبایی
 
 shop-greengrocer|@food
-en:4Greengrocer's|greengrocers|3grocery|U+1F345|U+1F346|U+1F33D|U+1F360|U+1F348|U+1F347|U+1F349|U+1F34A|U+1F34C|U+1F34D|U+1F34E|U+1F34F|U+1F350|U+1F351|U+1F353
+en:4Greengrocer|Grocer|greengrocers|3grocery|U+1F345|U+1F346|U+1F33D|U+1F360|U+1F348|U+1F347|U+1F349|U+1F34A|U+1F34C|U+1F34D|U+1F34E|U+1F34F|U+1F350|U+1F351|U+1F353
```

Comment:
The first name is capitalized because it's visible to users. Others are just search synonyms and visible to users only in the search field suggestions, so we don't capitalize them. A number before a word means "how many characters user should type in the search field to see the hint with a full word). So if you think people may search for grocer, it may make sense to add a number, for example, 4, so the full word "grocer" will appear after typing "groc". You may check how it works with other already present search synonyms.

Reply:
Understood. I'll remove the "Grocer", given the presence of '3grocery'.